dmoj/executors/CPP03.py

```diff
@@ -1,20 +1,20 @@
-from typing import Optional
-
 from .gcc_executor import GCCExecutor
 
 
 class Executor(GCCExecutor):
```

Comment:
The fact that the C++03 executor is used as a "cpp_executor" of sorts is pretty yucky; we should just make a dedicated one at some point.